spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/README.adoc

```diff
@@ -30,14 +20,12 @@ $ mvn spring-boot:run
 ----
 
 Browse to `http://localhost:8080`. This will trigger the `ExampleController.work()` method,
-which in turn will call other services, and also a remote RESTful call to `ExampleController.meet()`
-method.
+which in turn will call other services, and also a remote RESTful call to `ExampleController.meet()` method.
 
-To see the traces, navigate to Stackdriver Trace console's
-https://console.cloud.google.com/traces/traces[Trace List] view.
+To see the traces, navigate to Stackdriver Trace console's https://console.cloud.google.com/traces/traces[Trace List] view.
 You should see the trace information in detail.
+Additionally, if you logged in with the Google Cloud SDK or have the `GOOGLE_CLOUD_PROJECT` environment variable set to your GCP project ID, you can also click the `View` link in front of the `Details` -> `Log` to view the log entries related to that trace ID.
```

Comment:
This is really the only addition here. The rest is just putting sentences in the same line.